libs/graphics/spirv-tools/stone.yml

```diff
@@ -0,0 +1,28 @@
+#
+# SPDX-FileCopyrightText: © 2020-2023 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
+name        : spirv-tools
+version     : 2023.1
+release     : 1
+homepage    : https://github.com/KhronosGroup/SPIRV-Tools
+upstreams   :
+    - git|https://github.com/KhronosGroup/SPIRV-Tools.git : sdk-1.3.239.0
```

Comment:
Seems to build fine with `- https://github.com/KhronosGroup/SPIRV-Tools/archive/refs/tags/v2023.1.tar.gz: f3d8245aeb89f098c01dddaa566f9c0f2aab4a3d62a9020afaeb676b5e7e64d4` as the source. Any reason you didn't just use that?

Reply:
So I don't recall for certain but I remember in solus land there was a point that either the sdk-1.3.239.0 or 2023.1 tag wouldn't work probably. As phab is still down i can't check.